### DISCUSSION TITLE
Ability to add items to a nav bar menu

### DISCUSSION THREAD

**Discussion by benwoodward** [roam-garden/i#52] [Feature requests & Ideas]
If a reader lands on a page from a SERP, then I'd like for them to be able to click nav bar items to find their way around, e.g. "Contact", "About", "Home", "Essays".

I'd like to be able to specify these in a config file, or tag pages in roam so they show up in the nav bar, then perhaps use a Roam attribute to specify the menu link text.

**Comment by Stvad** [comment]
This makes sense! 

>then perhaps use a Roam attribute to specify the menu link text

This seems like something better suited to CSS? If I were to generate them with distinctive class/data tag would that work?

**Reply by benwoodward** [reply]
So you mean that you'd add a data-attribute to the menu items so I can add the menu text with CSS content, e.g.:

```css
a[data-nav="contact"]::after {
  content: "Contact Us";
}
```

That sounds like an elegant solution in terms of code, but I reckon it could be confusing for non-technical users. I'd be happy with this solution though.

**Reply by Stvad** [reply]
For some reason I read that as you requesting to specify link text color and not text content 🤦‍♂️  
Having a config for display name is totally valid!
